plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
BTW, we can get the disk number from this serial too: in the previous example FF4 is the 1st disk and its serial is SCES_038.40 while FF5 is the second and its serial is SCES_138.40.

The first digit of the serial + 1 is the disk number.